src/app/services/local-storage-manager.service.ts

```diff
@@ -15,8 +16,8 @@ export class LocalStorageManagerService {
     localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.TOKEN, loginObject.accessToken);
     localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.REFRESH_TOKEN, loginObject.refreshToken);
     localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.USERNAME, loginObject.username);
-    localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.TOKEN_EXPIRATION_DATE, String((decodedToken.exp - 10800) * 1000));
-    localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.TOKEN_CREATION_DATE, String((decodedToken.iat - 10800) * 1000));
+    localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.TOKEN_EXPIRATION_DATE, moment(((decodedToken.exp - 10800)) * 1000).toString());
+    localStorage.setItem(Constants.LOCAL_STORAGE.ACCESSORS.TOKEN_CREATION_DATE, moment((decodedToken.iat - 10800) * 1000).toString());
```

Comment:
Where your consts? If you see any value (but no 0 or 1), you should ask yourself: maybe need create separate const for this? ( 10800, 1000)